src/languages/asciidoc.js

```diff
@@ -113,12 +113,17 @@ export default function(hljs) {
         begin: /\*{2}[\w\s\n]+/,
         end: /\*{2}/
       },
+      // escaped formatting marks
+      {
+        begin: /\\/,
+        end: /[*_`]/
```

Comment:
This should all be a single `begin` regex... you don't want to match `\\blah blah blah*`.

Reply:
Initially that's what I did but I got the following exception:

```
Error: 0 width match regex
```

I'm now using `begin` with a `contains`. I added additional tests to cover these cases:

```
\\blah blah *blah*.

\\blah blah blah*.
```

Reply:
```
        begin: /\\[*_`]/,
```

Would work... `0 width` is only triggered when you have a rule that matches something 0 width, which is generally a no-no.

Reply:
Indeed it's working... I could have sworn that I tried that :flushed: